sdk/identity/identity/test/internal/node/azurePowerShellCredential.spec.ts

```diff
@@ -192,10 +191,9 @@ describe("AzurePowerShellCredential", function () {
     const credential = new AzurePowerShellCredential();
 
     const token = await credential.getToken(scope);
+    sandbox.restore();
```

Comment:
nit: `sandbox.restore()` should go in an afterEach block instead of each test.

It's easy to miss, and it doesn't get called if the test throws an error (unless you _also_ put it inside the `catch` or `finally` block) - it's simpler to have a single `sandbox.restore` in an `afterEach` block.

Reply:
done